### ISSUE TITLE
Two Fenster schließen in the Tab Nutzungsbedingungen

### ISSUE THREAD

**Issue by dpakprajul** [GDI-HE/GeoPortal.rlp#68]
There are two Fenster schließen link in the tab Nutzungsbedingungen. When tried on both, they the same functionality, i.e.  it closed the window (tab) in the browser.

### Example link: 
https://www.geoportal.hessen.de/mapbender/php/mod_showMetadata.php?languageCode=de&resource=wmc&layout=tabs&id=39
![grafik](https://github.com/GDI-HE/GeoPortal.rlp/assets/38970123/ba31d161-fd67-4ab4-99af-96461032720c)

When checked upon other tabs, only the upper Fenster schließen is present.
![grafik](https://github.com/GDI-HE/GeoPortal.rlp/assets/38970123/95551554-9b87-462b-8ccc-eb2f22c042be)

**Comment by Andreasr28** [comment]
For comparison, [see here](https://test.geoportal.hessen.de/mapbender/php/mod_showMetadata.php?languageCode=de&resource=layer&layout=tabs&id=47893)
The test system does not have this error.

**Comment by muqit-zoarder** [comment]
@Andreasr28 yes you are right. I have checked also in test system, and it has been corrected there for wmcs

**Comment by dpakprajul** [comment]
In the test system too, if you look in the **Karten**, it is still there.
 
![grafik](https://github.com/GDI-HE/GeoPortal.rlp/assets/38970123/730185ee-c487-425e-9e68-fd35299c2f34)

**Comment by Andreasr28** [comment]
Well that is the point. So far i can say this is only in "Karten". Could you please double check?

**Comment by dpakprajul** [comment]
I rechecked it. This is only in "Karten".

**Comment by dpakprajul** [comment]
I found one line of code: 
```html
echo "<div style='padding:10px;display:block;text-align:center;'><a href='javascript:window.close()'>Fenster schliessen</a></div>"; 
```
in mapbender/http/php/mod_getWmcDisclaimer.php in line 139. The "Karten" is not working in my test system. Could anybody try to remove it and see the result?

This is probably an issue of Mapbender. Could we move this issue to repo Mapbender2.8?

**Comment by Andreasr28** [comment]
@muqit-zoarder If you update the database of your vm then could you please show deepak how to do it?

**Comment by dpakprajul** [comment]
I have mentioned the problem in the mrmap-community/Mapbender2.8 too. https://github.com/mrmap-community/Mapbender2.8/issues/137

**Comment by karlbrink** [comment]
thx @dpakprajul
